vault/plugin_reload.go

```diff
@@ -77,8 +75,7 @@ func (c *Core) reloadMatchingPlugin(ctx context.Context, pluginName string) erro
 		if ns.ID != entry.Namespace().ID {
 			continue
 		}
-
-		if entry.Config.PluginName == pluginName && entry.Type == "plugin" {
+		if entry.Type == pluginName {
```

Comment:
What does entry.Type equal now?

Reply:
In the command, `$ vault auth enable my-custom-plugin`, the entry.Type == "my-custom-plugin".